Userland/Applications/Bits/BitsWidget.cpp

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2023, Pierre Delagrave <pierre.delagrave@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "BitsWidget.h"
+#include "GeneralTorrentInfoWidget.h"
+#include <AK/NumberFormat.h>
+#include <LibFileSystemAccessClient/Client.h>
+#include <LibGUI/Action.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/SortingProxyModel.h>
+#include <LibGUI/Splitter.h>
+
+int TorrentModel::column_count(GUI::ModelIndex const&) const
+{
+    return Column::__Count;
+}
+
+ErrorOr<String> TorrentModel::column_name(int column) const
+{
+    switch (column) {
+    case Column::Name:
+        return "Name"_short_string;
+    case Column::Size:
+        return "Size"_short_string;
+    case Column::State:
+        return "State"_string.release_value();
```

Comment:
As an example, you shouldn't use `release_value()` without checking that the result isn't an error.
You can either use `TRY()` if you want only the underlying value, or sometime (as in this situation) directly return the ErrorOr<String>.

Reply:
I've fixed all such cases, using `TRY` or explicitly checking for an error before releasing the value.
The only remaining `release_value()` are when using `HashMap::get()` and a value is expected to be there so a crash is a bug and not a situation we want to recover from.